src/main/resources/io/gatling/jenkins/tags/graph.jelly

```diff
@@ -25,14 +25,14 @@
 	</st:documentation>
 	<div id="${attrs.id}" style="width: ${attrs.width};height: ${attrs.height};">
 		<script type="text/javascript" language="javascript">
-			jQuery.noConflict();
-			jQuery(document).ready(function () {
-				var plot1 = jQuery.jqplot('${attrs.id}', ${attrs.data}, {
+			jQueryGatling.noConflict();
```

Comment:
Ah yeah, probably not. I think I had just done a find and replace for `jQuery`, but yeah I don't think there's any use in doing `noConflict` twice.